docs/UserGuide.md

```diff
@@ -97,6 +97,20 @@ Format: `adddebt <person index> d/<description> m/<money>`
 Example:
 * `adddebt 3 d/McDonalds m/8.9`
 
+### Splitting a debt: `splitdebt`
+
+Splits a debt among several people in PayMeLah such that each person owes the same amount.
```

Comment:
Maybe include the fact that it rounds up to the nearest cent for each person?

Reply:
Done, thanks!